recipes/pytorch-cpu/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pytorch-cpu
+  version: {{ version }}
+
+source:
+  # The tar source needs serious patching since it is trying to build
+  # all 3rd party modules
+  # url: https://github.com/pytorch/pytorch/archive/v{{ version }}.tar.gz
+  # fn: pytorch-v{{ version }}.tar.gz
+  # sha256: f360b7954e441bc5c4b36c49e711697a5c351de8b146c2982809385e3b3cfdcb
+  git_url: https://github.com/pytorch/pytorch.git
+  git_tag: v{{ version }}
+  patches:
+    - mac_osx_clock_getttime.patch  # [osx]
+
+# The build needs to set many environment variables
+# This is an easy way to do it cross platform
+{% set export = "export" %}  # [not win]
+{% set export = "set" %}     # [win]
+
+build:
+  number: 1000
+  # Needs modern GCC, the ones in conda-forge are too old
+  {% if 'toolchain' in compiler('cxx') %}skip: True{% endif %}  # Only build with modern compilers.
```

Comment:
I think this disabled the build of the package altogether in staged recipes.

Reply:
it will simply fail. It needs modern compilers, as well as few other PRs fixing old OSX stuff.

Reply:
<no judgement>Are you asking to import an empty feedstock so all build issues, including linux, osx, and win can be fixed outside staged-recipes? </no judgement>

If that is the case it might make it easier to simply say 
```yaml
build:
  number: 0
  skip: True 
```

Reply:
I'm not, I am asking to import a recipe that I tested mostly with conda-smithy tools that I rendered here: https://github.com/ramonaoptics/pytorch-cpu-feedstock

https://dev.azure.com/ramonaoptics/feedstock-builds/_build/results?buildId=13

There is a pretty high probability that it will work once the bot issues the compiler upgrade.

Reply:
To use new compilers on staged-recipes, add
```yaml
channel_sources:
- conda-forge/label/gcc7,defaults   # [unix]
- conda-forge,defaults              # [win]
channel_targets:
- conda-forge gcc7                  # [unix]
- conda-forge main                  # [win]
cxx_compiler:
- gxx      # [linux]
- clangxx  # [osx]
```

Reply:
Where should we add these?

Reply:
to `recipes/pytorch-cpu/conda_build_config.yaml`

Reply:
thanks for chiming in @isuruf!

Unfortunately, this doesn't solve any of the other outstanding issues here:
- The need for VS2017, which currently isn't being used in staged-recipes. This is a change in appveyor that needs to be done.
- The need to increase the timeout in travis.

If I do this, the only package that will be built is the linux one, which really isn't the "attractiveness" of conda-forge.

Please merge this package as is, so the rest of the work can be done in the feedstock.

Reply:
ok will do.